README.md

```
@@ -2,121 +2,213 @@
 
 # Agency Core
 
-### The autonomous AI platform for engineering teams — self-hosted, privacy-first, runs on your hardware.
+### The autonomous AI platform for engineering teams — self-hosted, privacy-first, runs anywhere.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)